### ISSUE TITLE
Compiler doesn't watch changes in partial files

### ISSUE THREAD

**Issue by Jeneko** [glenn2223/vscode-live-sass-compiler#10]
Sass doesn't compile, when do changes in partial files and use "liveSassCompile.settings.includeItems" settings (see: "[.vscode/settings.json](https://github.com/Jeneko/live-sass-compiler-bug-test/blob/main/.vscode/settings.json)"). It doesn't watch changes in partial files which imported in main scss file.

Here's an example code for illustration: [https://github.com/Jeneko/live-sass-compiler-bug-test](https://github.com/Jeneko/live-sass-compiler-bug-test). When do changes in **style.scss** all works well. But when do changes in **_some_styles.scss** nothing happens :(

**Comment by glenn2223** [comment]
Hi @Jeneko,

Thanks for reporting this bug.

I have found the problem [line 522](https://github.com/glenn2223/vscode-live-sass-compiler/blob/a93a22cf9487ccb866b53109216f0320a7ae7b2e/src/appModel.ts#L522). I have applied a fix and confirmed it works in my solution, plus the one you kindly included.

Keep an eye out for v4.3.1, being released momentarily.

Thanks again for making the extension better for everyone!
